### DISCUSSION TITLE
Imba 2.0 Reactivity Adapter

### DISCUSSION THREAD

**Discussion by ericvida** [maxnowack/signaldb#380] [General]
I build all my projects using [Imba](https://imba.io/).

I'll try to build a reactivity adapter, but I may need some help. I'll post my progress here.

**Comment by maxnowack** [comment]
I'm not familiar with Imba, but happy to help! Also take a look at the [other reactivity adapters](https://github.com/maxnowack/signaldb/tree/main/__tests__/reactivity) for inspiration 🙂
